README.md

```diff
@@ -16,7 +16,7 @@ processed.  We compute policy based on a global store which is distributed to Di
 
 Application Layer Policy is described in the [Project Calico docs][docs].
 
- - [Enabling Application Layer Policy](https://docs.projectcalico.org/master/getting-started/kubernetes/installation/app-layer-policy)
+ - [Enabling Application Layer Policy](https://docs.projectcalico.org/master/security/app-layer-policy)
  - [Application Layer Policy Tutorial](https://docs.projectcalico.org/latest/security/app-layer-policy/)
```

Comment:
Please can you also delete this "Application Layer Policy Tutorial" line, as it's now just another pointer to the same place.

Reply:
Since application layer policy tutorial was referring to latest stable version of the docs I thought it might be better to delete the other link.

Reply:
I'm sorry, I thought about that difference too, and reached the opposite conclusion.  I should have explained that in my previous response!

So .../master/... links to the master docs, and .../latest/... or just https://docs.projectcalico.org/security/app-layer-policy links to the latest release.  My thinking is that it's better for this README.md to link to master, because this README.md is itself a master doc.  WDYT?

Reply:
Honestly, Im now a bit confused. I was under the impression that "latest" is the stable version so we should refer users to that. However, since you have pointed out that this is the master branch I think it would be better to have link to "master" branch as well.